.github/workflows/build-ci-image.yml

```diff
@@ -11,13 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+---
```

Comment:
It's really bothering me that there are spaces being added here but given that it's on main I don't think we should be addressing that here.

Reply:
That's not a space, it's a `---` marker to signify the end of the yaml headers. It was added by one of our yaml linters.

Reply:
ooooh ok all good then!